hack/update-version-list.sh

```diff
@@ -66,5 +66,14 @@ function updateVersionsInCSVDescription() {
     rm "$tmpFile"
 }
 
+function updateVersionInSamples() {
+    defaultVersion=$(yq '.versions[0].name' "${VERSIONS_YAML_FILE}")
```

Comment:
This assumes the first entry is the default. Is this enforced in any way? Maybe adding a note in versions.yaml to state this while we don't have a not-so-fragile way of enforcing it?